fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -3581,7 +3581,7 @@ bool DDTeamCollection::satisfiesPolicy(const std::vector<Reference<TCServerInfo>
 	return result && resultEntries.size() == 0;
 }
 
-DDTeamCollection::DDTeamCollection(Database const& cx,
+DDTeamCollection::DDTeamCollection(const std::shared_ptr<IDDTxnProcessor>& dbProcessor,
```

Comment:
IMHO, it would be nice to call this just `db`.

Not picking on this change - I can see that this is a reasonable name and is relatively consistent with nomenclature standard in this codebase.

Just sharing that I'm reminded of this presentation whenever I choose names https://www.slideshare.net/Kevlin/giving-code-a-good-name. In particular slide 34 and 35 illustrate how our industry broadly tends to name things. Slide 29 and 30 illustrate the difference thinking like that can make to readability.

Reply:
Yeah. The `db` looks good to me. I'd like to rename together in another PR